gpii/node_modules/gpii-db-operation/src/DbDataStore.js

```diff
@@ -97,6 +97,48 @@ fluid.defaults("gpii.dbOperation.dbDataStore", {
                 }
             }
         },
+        expressUserSafeLoginLookup: {
+            type: "gpii.dbOperation.dbDataSource",
+            options: {
+                requestUrl: "/_design/views/_view/expressUserSafeLoginLookup?include_docs=true&key=%22%userRecordId%22",
+                termMap: {
+                    userRecordId: "%userRecordId"
```

Comment:
It would be nice to renamed the field name here ("userRecordId") with the actual field name "gpiiExpressUserId" defined in the document type.

Reply:
Fixed.